packages/manager/modules/pci/src/components/hds/template.html

```diff
@@ -0,0 +1,22 @@
+<div class="mb-3">
+    <oui-checkbox
+        data-disabled="!$ctrl.hds.isValidSupportLevel || !$ctrl.hds.isValidForCertification || $ctrl.hds.isCertifiedProject"
```

Comment:
I would have expected function as there is a `verb` but it's nitpicking

Reply:
My self i hesitate to create a function.
So, i created new one.

Reply:
fixed